src/platform/plugins/shared/dashboard/public/dashboard_actions/copy_to_dashboard_modal.tsx

```diff
@@ -51,18 +51,14 @@ export function CopyToDashboardModal({ api, closeModal }: CopyToDashboardModalPr
   const dashboardId = api.parentApi.savedObjectId$.value;
 
   const onSubmit = useCallback(() => {
-    const dashboard = api.parentApi;
+    const dashboard = api.parentApi as DashboardApi;
     const panelToCopy = dashboard.getDashboardPanelFromId(api.uuid);
-    const runtimeSnapshot = apiHasSnapshottableState(api) ? api.snapshotRuntimeState() : undefined;
-
-    if (!panelToCopy && !runtimeSnapshot) {
-      throw new PanelNotFoundError();
-    }
 
     const state: EmbeddablePackageState = {
       type: panelToCopy.type,
-      input: runtimeSnapshot ?? {
-        ...omit(panelToCopy.explicitInput, 'id'),
+      serializedState: {
+        rawState: { ...omit(panelToCopy.explicitInput, 'id') },
+        references: panelToCopy.references ?? [],
```

Comment:
Note that `getDashboardPanelFromId` also conditionally calls `snapshotRuntimeState` and throws a `PanelNotFound` error if unable to find any state for a given ID (either via the Dashboard panel state or via the child API) - the only difference in logic here would be that the serialized state is now spread on top of the panel state. But I think that's fine? 🤷